### ISSUE TITLE
Retune Tuner to sweet spots

### ISSUE THREAD

**Issue by Nokoa** [DSheirer/sdrtrunk#1277]
Often SDRTrunk may define a center frequency for the tuner causing some of the channels to be on the edges, while all channels can very well fit in the middle area of the tuner spectrum. The edges signal quality is degraded.

**Comment by GTR8000** [comment]
Yup, that's why some of us have taken to creating "dummy" channels to keep the centering optimized. I don't really like having extra channels running (even though they are decoding nothing), and it does make for a messy looking Now Playing screen, but it gets the job done.

Being able to define a static center channel per tuner would be a welcome feature.

**Comment by cfc62** [comment]
Would the dummy channels be set for the highest and lowest frequencies for each SDR?

**Comment by tvsjr** [comment]
@cfc62 correct. For example, an Airspy is good for 8.9875MHz of coverage (aliasing happens on the outside 1MHz, so SDRTrunk won't use it). Good frequency pairs (what I affectionately call Dummy Lo and Dummy Hi) are 767.50625 and 776.49375 for 700. 800 is a bit more challenging since the entire split is 10MHz (851-861). If you wanted to split the band in half, you could do something like 847.0125-856.0000 and 856.0125-865.0000.

IMO, especially considering the low price of the tuners, using a static center frequency is actually the optimal configuration. Retuning the tuner to try to maximize coverage is a klugey thing that should be the second choice for when you don't have/can't afford/whatever enough tuners to cover everything statically.
